### ISSUE TITLE
Rule request - whitespace around operators 

### ISSUE THREAD

**Issue by woodhamgh** [realm/SwiftLint#626]
Hi

It would be great if there was a rule for enforcing whitespace around operators.

For example the following would report an error
a+b
a+ b
a +b
let a=b

**Comment by norio-nomura** [comment]
Swift generate error on second and third examples:
```swift
  3> a+ b
repl.swift:3:3: error: consecutive statements on a line must be separated by ';'
a+ b
  ^
  ;

  3> a +b
repl.swift:3:2: error: consecutive statements on a line must be separated by ';'
a +b
 ^
 ;
```
So, this would be expected to generate violations on first and 4th examples.
